explorer/src/Pos/Explorer/Web/Server.hs

```diff
@@ -185,9 +191,10 @@ explorerHandlers _sendActions =
     getStatsTxsDefault page =
         catchExplorerError $ getStatsTxs page
 
-    defaultPageSize size = (fromIntegral $ fromMaybe 10 size)
-    defaultLimit limit   = (fromIntegral $ fromMaybe 10 limit)
-    defaultSkip  skip    = (fromIntegral $ fromMaybe 0  skip)
+    defaultPageSize size  = (fromIntegral $ fromMaybe 10 size)
+    defaultLimit limit    = (fromIntegral $ fromMaybe 10 limit)
+    defaultSkip skip      = (fromIntegral $ fromMaybe 0  skip)
+    defaultPage page      = (fromIntegral $ fromMaybe 1  page)
```

Comment:
Minor. Looks like a good candidate for Def typeclass.

Reply:
Yes, this was refactored and removed by @sectore in the `master` branch.